.github/CODEOWNERS

```
@@ -256,3 +256,5 @@
 /modules/xresources.nix                               @rycee
 
 /modules/xsession.nix                                 @rycee
+
+/modules/exa.nix                                      @kalhauge
```

I think the `CODEOWNERS` file is supposed to be alphabetically sorted so this should be underneath:
```
/modules/programs/emacs.nix
```

Also, the filename is incorrect.

```suggestion
/modules/programs/exa.nix                             @kalhauge
```

---

Good point!

Should be fixed.